engines/zvision/file/save_manager.cpp

```diff
@@ -129,61 +130,65 @@ void SaveManager::writeSaveGameHeader(Common::OutSaveFile *file, const Common::S
 
 Common::Error SaveManager::loadGame(int slot) {
 	Common::SeekableReadStream *saveFile = NULL;
-
-	if (slot >= 0) {
-		saveFile = getSlotFile(slot);
+	if (slot < 0) {
+		//Restart game, used by ZGI death screen only
+		/*
+		        saveFile = _engine->getSearchManager()->openFile("r.svr");
+		        if (!saveFile) {
+		            Common::File *restoreFile = new Common::File();
+		            if (!restoreFile->open("r.svr")) {
+		                delete restoreFile;
+		                return Common::kPathDoesNotExist;
+		            }
+		            saveFile = restoreFile;
+		        }
+		/*/
```

Comment:
Why has this been disabled? The game state is inconsistent after restarting without using `r.svr`. Has this change been tested?